packages/instrumenter/test/unit/mutators/arrow-function-mutator.spec.ts

```diff
@@ -19,4 +24,16 @@ describe(sut.name, () => {
   it('should not mutate an anonymous function with undefined as a body', () => {
     expectJSMutation(sut, 'const b = () => undefined');
   });
+
+  it('should only mutate what is defined in the mutator level', () => {
+    expectJSMutationWithLevel(sut, arrowFunctionLevel.ArrowFunction, 'const b = () => 4;', 'const b = () => undefined;');
+  });
+
+  it('should not mutate anything if there are no values in the mutation level', () => {
+    expectJSMutationWithLevel(sut, [], 'const b = () => 4;');
+  });
+
+  it('should mutate everything if the mutation level is undefined', () => {
+    expectJSMutationWithLevel(sut, arrowFunctionUndefinedLevel.ArrowFunction, 'const b = () => 4;', 'const b = () => undefined;');
+  });
```

Comment:
Since there is only one element, this test checks the same thing as `should only mutate what is defined in the mutator level` and can be deleted imo

Reply:
I (just now) mentioned in the other PR that I like to put all 4 in there, which is why I added this. I'd rather keep it, because undefined is a different case than having a defined level with something in it.